core/src/main/java/org/elasticsearch/search/SearchService.java

```diff
@@ -963,7 +963,7 @@ public boolean canMatch(ShardSearchRequest request) throws IOException {
     }
 
     /**
-     * Returns true iff the given search source builder can be early terminated by rewriting to a match none query. Or in other words
+     * Returns true if the given search source builder can be early terminated by rewriting to a match none query. Or in other words
```

Comment:
I think `iff` is intended here, meaning "if and only if" here (https://en.wikipedia.org/wiki/If_and_only_if), I'd leave this as is.